core/flyout.js

```diff
@@ -700,8 +700,9 @@ Blockly.Flyout.prototype.show = function(xmlList) {
         }
       } else if (tagName == 'BUTTON') {
         var label = xml.getAttribute('text');
+        var callback_key = xml.getAttribute('callbackKey');
```

Comment:
JS style guide advocates decapitatedCamelCase for variables rather than underscores.
https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming